bundles/cilium.v1.13.10/manifests/cilium.clusterserviceversion.yaml

```diff
@@ -0,0 +1,354 @@
+              metadata:
+                labels:
+                  name: cilium-olm
+              spec:
+                containers:
+                  - command:
+                      - /usr/local/bin/helm-operator
+                      - run
+                      - --watches-file=watches.yaml
+                      - --enable-leader-election
+                      - --leader-election-id=cilium-olm
+                      - --zap-devel
+                      - --metrics-addr=localhost:8082
+                      - --health-probe-bind-address=localhost:8081
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
+                      - name: RELATED_IMAGE_CILIUM
+                        value: quay.io/cilium/cilium@sha256:aff23a5d4daf06623d10f7ff6a8a7b617c22c1f8d63f507fd33ef3a80ba4779f
+                      - name: RELATED_IMAGE_HUBBLE_RELAY
+                        value: quay.io/cilium/hubble-relay@sha256:02702a6f1e3bbe4ebcd2fc5148e0f3f0fe170083170bd4109804ed5a269285c7
+                      - name: RELATED_IMAGE_CILIUM_OPERATOR
+                        value: quay.io/cilium/operator-generic@sha256:257a9fe37f0e689852cc899b32e5beae80ae574c7e3fcb9a6489973fe5301c99
+                      - name: RELATED_IMAGE_PREFLIGHT
+                        value: quay.io/cilium/cilium@sha256:aff23a5d4daf06623d10f7ff6a8a7b617c22c1f8d63f507fd33ef3a80ba4779f
+                      - name: RELATED_IMAGE_CLUSTERMESH
+                        value: quay.io/cilium/clustermesh-apiserver@sha256:e2049922fc00b6abe70ccdedf6b8130c80612ea1675a36d5b8492723d55134f5
+                      - name: RELATED_IMAGE_CERTGEN
+                        value: quay.io/cilium/certgen@sha256:f09fccb919d157fc0a83de20011738192a606250c0ee3238e3610b6cb06c0981
+                      - name: RELATED_IMAGE_HUBBLE_UI_BE
+                        value: quay.io/cilium/hubble-ui-backend@sha256:f88a73a120521eeafbcdbd908f517117f6557bf61e115847853fac371f0d774c
+                      - name: RELATED_IMAGE_HUBBLE_UI_FE
+                        value: quay.io/cilium/hubble-ui@sha256:e15af59a2ded739e420be82de6cbdd0ce22f8d3f00e3a10b3d2d2734e074a394
+                      - name: RELATED_IMAGE_ETCD_OPERATOR
+                        value: quay.io/cilium/cilium-etcd-operator@sha256:04b8327f7f992693c2cb483b999041ed8f92efc8e14f2a5f3ab95574a65ea2dc
+                      - name: RELATED_IMAGE_NODEINIT
+                        value: quay.io/cilium/startup-script@sha256:a1454ca1f93b69ecd2c43482c8e13dc418ae15e28a46009f5934300a20afbdba
+                      - name: RELATED_IMAGE_CLUSTERMESH_ETCD
+                        value: quay.io/coreos/etcd@sha256:a67fb152d4c53223e96e818420c37f11d05c2d92cf62c05ca5604066c37295e9
+                    image: registry.connect.redhat.com/isovalent/cilium-olm:7d87c60e3db562eec6101b16c9dc8cb0a4405130-v1.13.10
```

Comment:
shouldn't we have the digest here?

Reply:
Yes you are right according to the workflow in use:

> Once the image is published open a new PR in the [isovalent/certified-operators-cilium](https://github.com/isovalent/certified-operators-cilium), by adding the new manifests to the appropriately named new directory under operators/cilium (operators/cilium/v1.14.3, for example). Before commiting your changes make sure to modify the image reference in the manifests/cilium.clusterserviceversion.yaml so that the sha256 tag of the image is used (rather than the semantic tag).

Reply:
Actually, I will check whether we can get these from workflow run (the sha should be the same between registry.connect.redhat.com and quay.io right)?